ambry-commons/src/main/java/com/github/ambry/clustermap/HelixVcrUtil.java

```diff
@@ -43,10 +41,34 @@
  */
 public class HelixVcrUtil {
   private static final Logger logger = LoggerFactory.getLogger(HelixVcrUtil.class);
-  public static final List<String> ignoreResourceKeyWords = Arrays.asList("aggregation", "trigger", "stats");
   static final String SEPARATOR = "/";
   private static final ZNRecordSerializer ZN_RECORD_SERIALIZER = new ZNRecordSerializer();
   private static final String DELAYED_REBALANCER_CLASS_NAME = "DelayedAutoRebalancer";
+  private static final Set<String> smallIntegers = new HashSet<>();
+
+  static {
+    for (int i = 0; i < 128; i++) {
+      smallIntegers.add(String.valueOf(i));
+    }
+  }
+
+  /**
+   * Return {@code true} is the given resource name is a resource that contains partition information.
+   * @param resourceName The resource name.
```

Comment:
Please add a comment here saying something like "A resource is considered to be a partition resource only if the resource id is an integer.